src/routers/sessions.py

```
@@ -388,7 +388,12 @@ async def get_session_context(
     summary_tokens = int(token_limit * 0.4) if summary else 0
     messages_tokens = token_limit - summary_tokens
 
-    # Get the messages to return verbatim
+    logger.info(
+        f"Context request for session {session_id}: token_limit={token_limit}, "
+        + f"summary_tokens={summary_tokens}, messages_tokens={messages_tokens}, summary_requested={summary}"
```

Ruff (0.12.2)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)